GYBITG/NewAccountViewController.swift

```diff
@@ -0,0 +1,123 @@
+                self.dismiss(animated: true, completion: nil)
+    }
+    
+    @IBAction func createAccountTapped(_ sender: UIButton) {
+        
+        if (emailAddress.text == "") {
+            let alert = UIAlertController(title: "Alert", message: "Please enter an email address", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        } else if (password.text == "") {
+            let alert = UIAlertController(title: "Alert", message: "Please enter a password", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        } else if (password.text != passwordVerify.text) {
+            let alert = UIAlertController(title: "Alert", message: "Passwords do not match", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        } else if (firstName.text == "") {
+            let alert = UIAlertController(title: "Alert", message: "Please enter your first name", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        } else if (lastName.text == "") {
+            let alert = UIAlertController(title: "Alert", message: "Please enter your last name", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        } else if (highSchoolName.text == "") {
+            let alert = UIAlertController(title: "Alert", message: "Please enter your high school name", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        } else if (clubTeam.text == "") {
+            let alert = UIAlertController(title: "Alert", message: "Please enter your club team", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+        } else {
+            _ = usersRepository.addUser(userToAdd: User(userId: emailAddress.text!, password: password.text!, createDate: Date(), lastLoginDate: Date()))
+            self.dismiss(animated: true, completion: nil)
+        }
```

Comment:
Consider refactoring this repetitive code by extracting the differing portion of each section as a variable, and writing the common portion only once.